src/seedsigner/views/view.py

```diff
@@ -275,6 +272,7 @@ def run(self):
 
             # Give the screen just enough time to display the reset message before
             # exiting.
+            logger.info("Restarting SeedSigner")
```

Comment:
This is splitting the comment from the line of code it's explaining. I would move it above the comment.